homeassistant/components/comelit/coordinator.py

```diff
@@ -0,0 +1,52 @@
+"""Support for Comelit."""
+import asyncio
+from datetime import timedelta
+from typing import Any
+
+from aiocomelit import (
```

Comment:
I think this import will also fit on one line 🙈

Reply:
Black disagree with you ;-)

Reply:
damn 🙈 😄